File: Programers/Level1/자연수뒤집어배열로만들기.py
def solution(n):
    
    # str1 = str(n)
    str_list = list(str(n))
    str_list.reverse()
    return list(map(int, str_list))

# def solution(n):
    
#     str_list = list(str(n))
#     str_list.reverse()
#     answer = [0 for i in range(len(str_list))]
#     for i in range(len(str_list)) :
#         answer[i] = int(str_list[i])
        
#     return answer

File: Programers/Level1/자연수뒤집어배열로만들기.java
package Programers.Level1;

public class 자연수뒤집어배열로만들기 {
    
    public int[] solution(long n) {
        
        String str = Long.toString(n);
        int[] answer = new int[str.length()];
        
        for(int i=0; i < str.length(); i++){
            answer[i] = Integer.parseInt(str.charAt(str.length()-1-i)+"");
        }
        
        return answer;
    }

}
